### PR TITLE
ignore unobserved exceptions in V2FeedParser

### PR DESCRIPTION
**Pull request by campersau**
## Bug
Fixes: https://github.com/NuGet/Home/issues/7523
Regression: No  

## Fix
Details: explicitly ignore task exceptions to prevent them from going unobserved and crashing the process

## Testing/Validation
Tests Added: No  
Reason for not adding tests:  hard to test unobserved task exceptions

### CODE REVIEW

File: src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedParser.cs [hunk 1]
```diff
@@ -496,6 +496,14 @@ private static string GetString(XElement parent, XName childName)
                 results = results.Take(max).ToList();
             }
 
+            if (docRequest != null)
+            {
+                // explicitly ignore exception to prevent it from going unobserved
+                _ = docRequest.ContinueWith(t => { _ = t.Exception; },
```
Comment (campersau) [inline src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedParser.cs:502]: This is the actual fix. The other changes are for `.editorconfig`.

See https://blogs.msdn.microsoft.com/pfxteam/2009/06/01/tasks-and-unhandled-exceptions/

**Review by jainaashish — APPROVED** [review]

**Comment by clairernovotny** [comment]
Bump @nkolev92 any idea why the tests are failing?

**Comment by nkolev92** [comment]
@onovotny 
CI flakiness. Re-running it again, it might be helpful to rebase @campersau

**Comment by campersau** [comment]
@nkolev92 rebased

**Comment by clairernovotny** [comment]
@nkolev92 are the tests always this flaky? it'd be nice to finally get this merged in.

**Pull request merged**